README.md

```diff
@@ -3,132 +3,339 @@
+    > . ${NVM_DIR}/nvm.sh; nvm use v4.2.2; node ${NODE_DEBUG_OPTION} iCubTelemVizServer.js
+    
+    Now using node v4.2.2 (npm v2.14.7)
     iCub Telemetry server launched!
-    [INFO] |yarp.os.Port| Port /yarpjs/inertial:i active at tcp://192.168.1.18:10002/
+    [INFO] |yarp.os.Port| Port /yarpjs/inertial:i active at tcp://192.168.1.100:10117/
+    [INFO] |yarp.os.impl.PortCoreInputUnit| Receiving input from /icubSim/inertial to /yarpjs/inertial:i using tcp
+    ...
+    [INFO] |yarp.os.Port| Port /yarpjs/sysCmdsGenerator/rpc active at tcp://192.168.1.100:10124/
+    OK
+    Opem-MCT static server process started.
     ICubTelemetry History hosted at http://localhost:8081/history
     ICubTelemetry Realtime hosted at ws://localhost:8081/realtime
-    listening on *:3000
-    ```
-5. On the same machine, run `npm start` from `<yarp-openmct-root-folder>/openmctStaticServer` folder. You should get the following output on the terminal:
+    listening on http://localhost:3000
+    [OPEN-MCT STATIC SERVER] stdout:
+    > openmctStaticServer@1.0.0 start <user-home>/dev/yarp-openmct/openmctStaticServer
+    > . ${NVM_DIR}/nvm.sh; nvm use v14.17.0; node server.js
+    
+    
+    [OPEN-MCT STATIC SERVER] stdout: Now using node v14.17.0 (npm v6.14.13)
+    
+    [OPEN-MCT STATIC SERVER] stdout: iCub Telemetry Visualizer (Open MCT based) hosted at http://localhost:8080
     ```
-    > openmctStaticServer@1.0.0 start <yarp-openmct-root-folder>/openmctStaticServer
-    > node server.js
 
-    iCub Telemetry Visualizer (Open MCT based) hosted at http://localhost:8080
-    ```
-    The Open MCT visualizer static server creates the main interface page and sends it to the Open MCT client.
-6. Read the device IP address using `ifconfig` (Linux,MacOS) or `ipconfig` (Windows). We shall refer to this address as <server-IP-address>.
+4. Read the machine IP address using `ifconfig` (Linux,MacOS) or `ipconfig` (Windows). We shall refer to this address as \<server-IP-address>.
+
+## [How to Run the Visualizer Client](#top)
 
-## [How to Run the Client](#top)
+The Visualizer Client is a GUI based on the [Open MCT](https://github.com/nasa/openmct) framework, displaying a set of iCub Telemetry data elements which plot the data received from the telemetry server. 
 
-Open a browser on any other device connected to the same network and go to `<server-IP-address>:8080`.
+Run a browser on any other machine connected to the same network and open the link `<server-IP-address>:8080`. If you run the browser on the same machine as the telemetry server, just open the link `localhost:8080`.
 
 <p align='center'>
 <img src="images/iCubTelemetryOpenMCTexample.png" width="80%">
 </p>
 
+In the above example, the iCub head IMU measurements read on the port `/icub/inertial` and sent to the visualization tool client are exposed as the telemetry node "IMU sensor measurements". The telemetry node wraps all the 12 measurement components:
```

Comment:
Actually, above is ``/icubSim/inertial``

Reply:
ah yep! updated.